eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -154,8 +157,12 @@ public void onError(String errorMessage) {
                 })
                 .buildAndInit();
 
-        if(!lock.await(10000, TimeUnit.MILLISECONDS)) {
-            throw new RuntimeException("Request for RAC did not complete within timeout");
+        try {
+            if (!lock.await(10000, TimeUnit.MILLISECONDS)) {
+                throw new RuntimeException("Request for RAC did not complete within timeout");
+            }
+        } catch (InterruptedException e) {
```

Comment:
By handling `InterruptedException` here and removing the `throws` from the signature, it cleaned up our test code a decent amount. 🧹